openpower/patches/stonewither-patches/linux/0001-power-fixed2.patch

```diff
@@ -0,0 +1,17 @@
+--- a/drivers/net/ethernet/intel/fm10k/fm10k_pci.c	2018-01-10 15:27:42.437514511 -0600
```

Comment:
We like kernel patches to go into upstream kernels and we pick them up through the stable tree (we consider any local patches we're carrying to be a bug :)

Has this one been submitted to upstream stable?

If we need something before the next stable kernel is released, we'll pick it up, but globally for op-build so that we don't end up with many kernel trees to maintain.